<launch>
    <node name="dynamixel_workbench" pkg="dynamixel_workbench" type="dynamixel_workbench" output="screen">
        <param name="port" value="/dev/ttyUSB0" />  <!-- 根据您的设置更改端口 -->
        <param name="baud_rate" value="57600" />
    </node>

    <node name="dynamixel_controller" pkg="dynamixel_workbench_controllers" type="dynamixel_controller" output="screen">
        <param name="joint_name" value="joint_1" />  <!-- 根据您的舵机名称更改 -->
        <param name="control_mode" value="position" />  <!-- 控制模式 -->
    </node>
</launch>